tester/main.cpp

```diff
@@ -307,26 +307,26 @@ int main(int argc, char const *argv[]) {
 			for (const auto &k : cat.kernel_config_list) {
 				for (const auto &datatype : k.data_type_option_list) {
 					const std::string label = cat.category_label + " " +
-					                          k.kernel_label + " - " +
-					                          size.first + " - " +
-					                          datatype.first;
+						k.kernel_label + " - " +
+						size.first + " - " +
+						datatype.first;
 					builder._compiler = default_compiler;
 					const std::string kernelSourceDir = polybench_source + "/" +
-					                                    cat.category_folder + "/" +
-					                                    k.kernel_folder;
-					vc::Option sourceDir("sourceDir", "-I", kernelSourceDir);
-					builder._optionList = {size.second, datatype.second, sourceDir};
-					builder._optionList.insert( builder._optionList.end(),
-					                            default_options.begin(),
-					                            default_options.end());
-					builder._fileName_src = kernelSourceDir + "/" + k.fileName;
-					builder._functionName = k.function_name;
+						cat.category_folder + "/" +
+						k.kernel_folder;
+					builder._optionList = {
+						vc::make_option("-I" + kernelSourceDir),
+						size.second,
+						datatype.second,
+					};
+					builder._optionList.insert(builder._optionList.end(), default_options.begin(), default_options.end());
+					builder._fileName_src = {kernelSourceDir + "/" + k.fileName, polybench_source + "/utilities/polybench.c"};
+					builder._functionName = {k.function_name};
```

Comment:
this is definitely OK and should work.

However, libVC version 2 provides additional setters for the builder class, which are a bit more OOP-like.
See, e.g. `vc::Version::Builder::options()`, `vc::Version::Builder::addIncludeDir()` and `vc::Version::Builder::addSourceFile()`.

Can you see if using them would improve the code readability?